hal/src/sercom/v2/uart/reg.rs

```diff
@@ -74,8 +89,11 @@ impl<S: Sercom> Registers<S> {
     #[inline]
     pub(super) fn configure_pads(&mut self, rxpo: u8, txpo: u8) {
         self.usart().ctrla.modify(|_, w| unsafe {
-            w.rxpo().bits(rxpo);
-            w.txpo().bits(txpo)
+            #[cfg(feature = "samd20")]
+            w.txpo().bit(txpo != 0);
+            #[cfg(not(feature = "samd20"))]
+            w.txpo().bits(txpo);
+            w.rxpo().bits(rxpo)
```

Comment:
We should review the differences in SERCOM carefully. @jbeaurivage, have you looked at this yet?

Reply:
I haven't yet, but I'd definitely like to comb through this finely with the datasheet next to me. Annoyingly enough, it seems the SAMD20 isn't exactly 'just a SAMD21 minus the USB'.

Reply:
@didjcodt, can you please explain what's the reasoning here? This kind of thing should probably be handled at the `impl_rxpotxpo` level rather than inside the functions.